processor/transformprocessor/config_test.go

```diff
@@ -122,7 +127,13 @@ func TestLoadConfig(t *testing.T) {
 			assert.NoError(t, component.UnmarshalConfig(sub, cfg))
 
 			if tt.expected == nil {
-				assert.Error(t, component.ValidateConfig(cfg))
+				err = component.ValidateConfig(cfg)
+				assert.Error(t, err)
+
+				if tt.errorLen > 0 {
```

Comment:
Pardon my ignorance here, but this PR introduces `errorLen` as a new variable, but it's never set or used in any of the test cases (that I can tell). Is this just here for future test case usages, or am I missing something?

Reply:
You're not missing anything, I just totally forgot to actually specify the value in my test case. It's fixed now, thanks.